### PR TITLE
fix(import): Fix concurrentModification error - issue #158

### PR DESCRIPTION
**Pull request by junajan**
This PR fixes an issue logged here https://github.com/sphereio/sphere-node-product-csv-sync/issues/153 and here
https://github.com/commercetools/express-impex/issues/158#issuecomment-274776454

This error occurs when there are multiple variants of the same product updated concurrently so I added another process which will retry request when a concurrentModification occurs.

### CODE REVIEW

**Comment by hisabimbola** [comment]
@junajan can you create an issue about this in this repo and reference this PR to close both the issue and the referenced issue in the impex repo

File: src/coffee/import.coffee [hunk 1]
```diff
@@ -294,18 +303,17 @@ class Import
 
   createOrUpdate: (products, types, matchFn) ->
     Promise.all _.map products, (entry) =>
-      @repeater.execute =>
```
Comment (hisabimbola) [inline src/coffee/import.coffee]: why are we removing this?

Comment (junajan) [inline src/coffee/import.coffee]: Repeater was there to handle HTTP errors which are now handled already in `sphere-node-sdk` module: https://github.com/sphereio/sphere-node-sdk/releases/tag/v1.17.0

**Comment by junajan** [comment]
@hisabimbola - I added issue in this repository and updated version of node-sphere-sdk module which repeats requests after HTTP error by default.

**Comment by junajan** [comment]
@hisabimbola - I added release draft here: https://github.com/sphereio/sphere-node-product-csv-sync/releases/edit/untagged-719e1d68fe1e66926d64

**Pull request merged**